settings.gradle.kts

```diff
@@ -1,3 +1,4 @@
+@file:Suppress("UnstableApiUsage")
```

Comment:
On my computer it looks like this is not needed.

Reply:
update to the newest version of android studio and get a lot of warnings about stuff being marked @incubating see above comment